src/index.ts

```diff
@@ -332,8 +332,13 @@ class SOOSSCAAnalysis {
       const exitCode = getExitCodeFromStatus(scanStatus);
       if (exitCode > 0 && this.args.onFailure === OnFailure.Fail) {
         soosLogger.warn("Failing the build.");
+        exit(exitCode);
+      } else if (exitCode === 2 && this.args.onFailure === OnFailure.Continue) {
+        soosLogger.warn("Issues found but continuing the build.");
+        if (this.args.integrationName === IntegrationName.AzureDevOps) {
+          exit(exitCode);
```

Comment:
only throw an exit code 2 when the integration is azure devops, rest of them dont have custom logic to just warn